src/kubectl.ts

```diff
@@ -272,3 +279,8 @@ function path(context: Context): string {
     const bin = baseKubectlPath(context);
     return binutil.execPath(context.shell, bin);
 }
+
+function getKubectlArgs(context: Context): string {
+    const skipTlsVerification: boolean = context.host.getConfiguration('vs-kubernetes')[`vs-kubernetes.skip-tls-verification`];
+    return `--insecure-skip-tls-verify=${skipTlsVerification}`;
```

Comment:
We often display our generated `kubectl` commands (e.g. when running in a terminal) so consider returning the empty string when the setting is false (to reduce noise in the displayed command).